cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/MultivariateAnamolyDetectionSuite.scala

```diff
@@ -36,7 +38,13 @@ object MADListModelsProtocol extends DefaultJsonProtocol {
 
 import com.microsoft.azure.synapse.ml.cognitive.split1.MADListModelsProtocol._
 
-object MADUtils extends AnomalyKey {
+trait storageCredentials {
+  lazy val connectionString: String = sys.env.getOrElse("STORAGE_CONNECTION_STRING", Secrets.StorageConnectionString)
+  lazy val storageKey: String = sys.env.getOrElse("STORAGE_KEY", Secrets.StorageKey)
```

Comment:
Then I guess I'll create another storage account, because in order to support using `endpoint and sas token` instead of connectionString, we need storage key to authenticate the client thus it could generate sas token afterwards.

Reply:
Or could we just support authenticate through connection string? It's way more simpler than the endpoint and sas token one:
`
def this(storageConnectionString: String, containerName: String) =
    this {
      val blobContainerClient = new BlobServiceClientBuilder()
        .connectionString(storageConnectionString)
        .credential(StorageSharedKeyCredential.fromConnectionString(storageConnectionString))
        .buildClient()
        .getBlobContainerClient(containerName.toLowerCase())
      if (!blobContainerClient.exists()) {
        blobContainerClient.create()
      }
      blobContainerClient
    }

  def this(storageName: String, storageKey: String, endpoint: String, sasToken: String, containerName: String) =
    this {
      val blobContainerClient = new BlobServiceClientBuilder()
        .endpoint(endpoint)
        .sasToken(sasToken)
        .credential(new StorageSharedKeyCredential(storageName, storageKey))
        .buildClient()
        .getBlobContainerClient(containerName.toLowerCase())
      if (!blobContainerClient.exists()) {
        blobContainerClient.create()
      }
      blobContainerClient
    }
`